data/gegno/power.txt

```diff
@@ -0,0 +1,119 @@
+outfit "R01 Skirmish Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "R01 Skirmish Batteries"
+	cost 17500
+	mass 12
+	thumbnail outfit/gbatterytiny
+	"energy capacity" 1121
+	"outfit space" -12
+	description "Skirmish batteries are very common amongst ships across Gegno space. They can be either a perfectly convenient storage of battery power to offset the load of larger Gegno generators, or a nice addition to add on top of them where more energy would be needed."
+
+outfit "R02 Battlezone Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "R02 Battlezone Batteries"
+	cost 32000
+	mass 23
+	thumbnail outfit/gbatterysmall
+	"energy capacity" 2237
+	"outfit space" -23
+	description "Battlezone batteries are primarily used on the Gegno's medium-range combat vessels, holding a decent amount of power for long lasting engagements. They also fit nicely on larger vessels as an extra power source to feed their larger systems."
+
+outfit "R03 Warforge Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "R03 Warforge Batteries"
+	cost 61700
+	mass 41
+	thumbnail outfit/gbatterymed
+	"energy capacity" 4469
+	"outfit space" -41
+	description "Warforge batteries have proved their usefulness over the years as large, sturdy battery packs. They support the equally large volleys that are fired from Gegno ships, and help maintain an excess of power for engine usage."
+
+outfit "R04 Crusade Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	plural "R04 Crusade Batteries"
+	cost 136000
+	mass 83
+	thumbnail outfit/gbatterylarge
+	"energy capacity" 8885
+	"outfit space" -83
+	description "As the largest battery mounted on Gegno ships, the Crusade Battery supplies almost enough power as a small generator, enough to support the massive warships of both the Scin and the Vi. However, they fall under the capacity of similarly-sized human batteries due to inefficient handling of their scale."
+
+outfit "C3 Scrap Cell"
+	category Power
+	licenses
+		"Gegno Civilian"
+	cost 54000
+	mass 25
+	thumbnail outfit/vicoretiny
+	"energy generation" 1.4
+	"heat generation" 2.2
+	"outfit space" -25
+	description "The Scrap Cell is a small radiothermal generator that produces energy generation in a pinch that abusing the Gegno's slightly hotter running tech. Its output is very limited, and is only used in tandem with the Gegno's smaller-sized systems."
+
+outfit "C7 Brawl Core"
+	category Power
+	licenses
+		"Gegno Civilian"
+	cost 68100
+	mass 39
+	thumbnail outfit/vicoresmall
+	"energy generation" 2.5
+	"heat generation" 3.9
+	"outfit space" -39
+	description "C7 Brawl Cores are a slight deviation from the other C class cores as it is set upright-forward. It is a more rudimentary type of radiothermal reactor than the C3, but nevertheless produces the necessary power for the ships that typically carry one."
```

Comment:
Rudimentary was an attempt at me to explain the reactor is more old in design (at least by irl human standards- instead of an upright reactor it resembles more of a sideways generator. Maybe this comparison doesn't make sense as there are too many types of generators to support this.) Will think of a different way to explain it.

Reply:
`C7 Brawl Cores are a slight deviation from the other C class cores as it is set upright-forward, a more rudimentary configuration. The tried and tested design, however, still serves the Gegno well, though it will likely be phased out when they are better able to scale up newer technologies.`

Reply:
Sounds good.